tests/test_calls_recording.py

```diff
@@ -0,0 +1,44 @@
+
+import pytest
+import requests
+
+
+@pytest.fixture
+def endpoint_url(httpserver):
```

Comment:
this function says nothing about adding a new request handler to httpserver. why is it extracted to a fixture? I think there's no need to make the tests more complicated, copying these lines to the test functions are better as it helps to understand what the tests are doing